tests/data/virtual-server-dos/README.md

```diff
@@ -0,0 +1,76 @@
+# DOS
```

Comment:
this file looks out of place. seems like it was copied by mistake

Reply:
removed